resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/tagged/TaggedCircuitBreakerMetrics.java

```diff
@@ -63,7 +63,7 @@ public static TaggedCircuitBreakerMetrics ofCircuitBreakerRegistry(MetricNames m
     @Override
     public void bindTo(MeterRegistry registry) {
         for (CircuitBreaker circuitBreaker : circuitBreakerRegistry.getAllCircuitBreakers()) {
-            addMetrics(registry, circuitBreaker);
+            addMetrics(registry, circuitBreaker, circuitBreakerRegistry);
```

Comment:
Is it needed? 
If instance(CircuitBreaker) is made from registry, instance has not only instance's tag but also registry's tags. So I think it can be `addMetrics(registry, circuitBreaker)`.

What do you think?

Reply:
How do we proceed?

Reply:
it is not needed , code cleaned that was leftovers from first solution approach to merge all tags